numpy/ma/tests/test_extras.py

```diff
@@ -1580,9 +1580,9 @@ def test_atleast_2d(self):
         assert_equal(b.mask.shape, b.data.shape)
 
     def test_shape_scalar(self):
-        # the atleast and diagflat function should work with scalars
+        # the at least and diagflat function should work with scalars
```

Comment:
`atleast` is a function name, so it should not be corrected. More precisely, it turns up as `atleast_(1d|2d|3d)`

Reply:
Did as you said, waiting for review :).